src/components/HomePageLayout/HomePageLayout.tsx

```diff
@@ -19,13 +20,26 @@ export const HomePageLayout: React.FC = () => {
       })
       .catch(error => console.error('Error fetching phones:', error));
   }, []);
+
+  useEffect(() => {
+    getProducts('./api/phones.json')
+      .then(phones => {
+        phones.sort((a, b) => a.priceRegular - b.priceRegular);
+        const limitedPhones = phones.slice(0, 12);
+        setNewModels(limitedPhones);
+      })
+      .catch(error => console.error('Error fetching phones:', error));
+  }, []);
+
```

Comment:
You can use one fetch to get phones 
After fetch: 
- newModels just save as it is 
- hotPrices sort and save